### PR TITLE
Reset subsystem

### PR DESCRIPTION
**Pull request by Rohan-Sreenivasan**
This allows us to stop all the motors for safety.

### CODE REVIEW

File: src/main/java/frc/team166/robot/subsystems/Drive.java [hunk 1]
```diff
@@ -185,7 +190,7 @@ protected boolean isFinished() {
                 if (frontLidar.getDistance(true) <= inches) {
                     return true;
                 } else {
-                    return false;
+                return false;
```
Comment (bot190) [inline src/main/java/frc/team166/robot/subsystems/Drive.java]: We shouldn't remove this tab.

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java [hunk 2]
```diff
@@ -78,7 +78,7 @@ public Manipulator() {
         SmartDashboard.putData("Deploy Manipulator With Joystick", DeployManipulatorWithJoystick());
         SmartDashboard.putData("Re-Enable Potentiometer", enablePID());
 
-        // Preferences Are Wanted In The Constructer So They Can Appear On Live Window
+        //Preferences Are Wanted In The Constructer So They Can Appear On Live Window
```
Comment (bot190) [inline src/main/java/frc/team166/robot/subsystems/Manipulator.java]: This space shouldn't be removed

File: src/main/java/frc/team166/robot/Robot.java [hunk 3]
```diff
@@ -54,6 +55,9 @@ public void robotInit() {
      */
     @Override
     public void disabledInit() {
+        drive.reset();
+        // lift.reset();
```
Comment (ghost) [inline src/main/java/frc/team166/robot/Robot.java]: Why are we not including the lift as part of this?

File: src/main/java/frc/team166/robot/subsystems/Drive.java (hunk 1 above)
Comment (ghost) [inline src/main/java/frc/team166/robot/subsystems/Drive.java]: ...removing indentation? Why??

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java [hunk 4]
```diff
@@ -143,7 +146,7 @@ private void setMotorsToDischarge() {
         // change once you find optimal motor speed
     }
 
-    // COMMANDS
+    //COMMANDS
```
Comment (bot190) [inline src/main/java/frc/team166/robot/subsystems/Manipulator.java]: Same here. Guessing this was caused by a bad merge.

**Review by bot190 — COMMENTED** [review]
Looking good, just a few nitpicks.

File: src/main/java/frc/team166/robot/subsystems/Manipulator.java [hunk 5]
```diff
@@ -93,6 +93,9 @@ public Manipulator() {
     }
 
     // METHODS  
+    public void reset() {
+        rollers.stopMotor();
```
Comment (ghost) [inline src/main/java/frc/team166/robot/subsystems/Manipulator.java:97]: I believe there are more motors than just the rollers (deploymentMotor)

**Review by ghost — CHANGES_REQUESTED** [review]

File: src/main/java/frc/team166/robot/Robot.java (hunk 3 above)
Comment (bot190) [reply src/main/java/frc/team166/robot/Robot.java]: We don't actually create a lift object in robot yet.

**Pull request merged**